web/src/components/predictions-components/multiple-category-choice.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from "react";
+import { Box, Label } from "theme-ui";
+import { Range, getTrackBackground } from 'react-range';
+
+const STEP = 0.1;
+
+class MultipleCategoryChoice extends Component {
```

Comment:
Could you try writing this component as a function with [useState](https://reactjs.org/docs/hooks-state.html)?

Reply:
Yep, I can check that out